vllm/model_executor/models/__init__.py

```diff
@@ -74,6 +78,8 @@ class ModelRegistry:
 
     @staticmethod
     def load_model_cls(model_arch: str) -> Optional[Type[nn.Module]]:
+        if model_arch in _OOT_MODELS:
```

Comment:
Can we print a warning when `model_arch` exists in both registries?

Reply:
fixed in [c3a3b16](https://github.com/vllm-project/vllm/pull/3871/commits/c3a3b16b7f67ede7b40414a90a3c4a6055781330)